Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.6-onbuild
```

Comment:
If you run `make build` for building binary, base image should be `golang:1.6`. `-onbuild` image does not use `glide` for download dependencies.